.circleci/config.yml

```diff
@@ -88,6 +88,8 @@ jobs:
           environment:
             MLFLOW_PIPELINES_PROFILE: local
           command: |
+            # Temporary patch for custom_metrics.py to make it return a scalar value
```

Comment:
I think we also need to file a PR against https://github.com/mlflow/mlp-regression-template#custom-metrics to update the README indicating that custom metric functions return scalars instead of dicts. We also need to file a PR against https://github.com/mlflow/mlp-regression-example to update the example custom metric functions. Can you file these PRs and file a JIRA ticket to track merging them right before the MLflow 2.0 release?

Reply:
Filed these two PRs:
- https://github.com/mlflow/mlp-regression-template/pull/98
- https://github.com/mlflow/mlp-regression-example/pull/18


Will file a Jira